README.md

```diff
@@ -65,6 +65,13 @@ The software behind Wikipedia
 * license: CC BY-SA 3.0
 * [website](https://www.mediawiki.org/wiki/MediaWiki) | [repo](https://phabricator.wikimedia.org/diffusion/)
 
+### SignDict
+SignDict is a english and german dictionary for sign language (using the GSL and DGS respectively). Currently it has over 4000 signs listed.
```

Comment:
I believe GSL and DGS are the same thing (Deutsh ...  and German Sign Language).
Can't find the GSL acronym in the wiki page: https://en.wikipedia.org/wiki/Sign_language

Reply:
As a side note, DGS doesn't seem that popular compared to other sign languages.
https://en.wikipedia.org/wiki/List_of_sign_languages_by_number_of_native_signers
Indo-Pakistani Sign Language is the most used one, but that's a bit unfair given the sheer number of indians out there. I'm wondering if there isn't an equivalent to English in the sign languages world.

Reply:
You're right, I didn't even think about that, sorry.

I'll update the description.

About the english equivalent of sign languages: It seems that there isn't an universally established one (yet).